### ISSUE TITLE
end - 1 char

### ISSUE THREAD

**Issue by Akuli** [Akuli/porcupine#475]
```
<Akuli> "end - 1 char" is a tkinter weirdness
<Akuli> you would expect "end" to mean the end
<Akuli> but actually, "end" is one character BEYOND the end, and the character is always \n
<Akuli> so "end - 1 char" means the end
<Akuli> - as in minus as in go back
<ThePhilgrim> ah got it
```

Would be nice to have this in docs

**Comment by ThePhilgrim** [comment]
You don't really have a section in the README about tkinter weirdness. Where in the README would you think this is fitting?

**Comment by Akuli** [comment]
"Potential Pitfalls" in docs/plugin-intro.rst

**Comment by Akuli** [comment]
Fixed in 6b67113a84e7cf985ad46cf4a85667f384ecf950.
